charts/4allportal/templates/webdav/deployment.yaml

```diff
@@ -144,7 +146,24 @@ spec:
             timeoutSeconds: {{ $.Values.webdav.readinessProbe.timeoutSeconds }}
           {{- end }}
           resources: {{- toYaml $.Values.webdav.resources | nindent 12 }}
+        - name: shared-dav
+          image: quay.io/curl/curl
+          volumeMounts:
+            - name: shared-dav-events
+              mountPath: /dav-events
+            - name: scripts
+              subPath: log-handler.sh
+              mountPath: /usr/bin/log-handler.sh
+              readOnly: true
+          command: [ "/bin/sh" ]
+          args: [ "-c", "tail -f /dav-events/logfile | ash /usr/bin/log-handler.sh" ]
       volumes:
+        - name: shared-dav-events
+          emptyDir: { }
```

Comment:
For stability reasons, we should set a size limit. Also, setting "Memory" can not hurt:

For example like this:
```yaml
         emptyDir:
           medium: Memory
           sizeLimit: 10Mi
```

Reply:
Maybe also add a file size limit to `rotatelogs` beside your time limit. Just to make sure, if someone is stress testing or doing bulk operations, we never run out of space.

Reply:
Done.
Apache `rotatelogs` configured with 1Mb file size limit. Also instead of `rotating` we use `truncation`, when only one log file present in same time (`-t` parameter, `tail` friendly). Apache log file created immediatly so `shared-dav` container will start right after `webdav` without errors (`-f` & `-c` parameters). 

`CustomLog "| /usr/local/apache2/bin/rotatelogs -t -f -c /dav-events/logfile 1M" "%s %m %U \"%{Destination}i\""`